test/e2e/performance/bpmn.load.performance.test.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { BpmnDiagramPreparation, BpmnLoadMethod, getSimplePlatformName, PageTester } from '../helpers/visu-utils';
+import * as fs from 'fs';
+import { calculateMetrics, ChartData } from '../helpers/perf-utils';
+import { FitType } from '../../../src/component/Options';
+
+interface PerformanceMetric {
+  run: number;
+  TaskDuration: number;
+  ScriptDuration: number;
+  RecalcStyleDuration: number;
+  LayoutDuration: number;
+}
+
+const platform = getSimplePlatformName();
+const performanceDataFilePath = './performance/data/' + platform + '/data.js';
+const metricsArray: Array<PerformanceMetric> = [];
+
+describe.each([1, 2, 3, 4, 5])('load performance', run => {
+  // to have mouse pointer visible during headless test - add 'showMousePointer=true' to queryParams
+
+  const fileName = 'B.2.0';
+
+  it.each([1])('check performance for file loading and displaying diagram with FitType.Center', async () => {
+    const metricsStart = await page.metrics();
+    const bpmnDiagramPreparation = new BpmnDiagramPreparation(new Map([['B.2.0', BpmnLoadMethod.Url]]), { name: 'navigation-diagram', queryParams: [] }, 'performance', {
+      fit: { type: FitType.Center },
+    });
```

Comment:
We cannot put `Center` for now because tests results are related to the default `FitType` which is `HorizontalVertical`

```suggestion
  it.each([1])('check performance for file loading and displaying diagram with FitType.HorizontalVertical', async () => {
    const metricsStart = await page.metrics();
    const bpmnDiagramPreparation = new BpmnDiagramPreparation(new Map([['B.2.0', BpmnLoadMethod.Url]]), { name: 'navigation-diagram', queryParams: [] }, 'performance', {
      fit: { type: FitType.HorizontalVertical },
    });
```